contrib/elastic/go-elasticsearch.v5/elastictrace_test.go

```diff
@@ -0,0 +1,167 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package elastic
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+)
+
+const debug = false
+
+const (
+	elasticV5URL = "http://127.0.0.1:9201"
+	elasticV6URL = "http://127.0.0.1:9202"
+	elasticV7URL = "http://127.0.0.1:9203"
+)
+
+func TestMain(m *testing.M) {
+	// _, ok := os.LookupEnv("INTEGRATION")
+	// if !ok {
+	// 	fmt.Println("--- SKIP: to enable integration test, set the INTEGRATION environment variable")
+	// 	os.Exit(0)
+	// }
```

Comment:
Why is this commented?

I think we want the check for `"INTEGRATION"` since these tests depend on the elasticsearch container, right?